src/sentry/web/forms/add_project.py

```diff
@@ -1,49 +0,0 @@
-from __future__ import absolute_import
```

Comment:
Why is this being deleted? Seems unrelated.

Reply:
yeah it was removed here https://github.com/getsentry/sentry/commit/d95c3ebcf219b99ff95ea48cda00476d4597d1f8#diff-615ecf8b7cfbfd9bf1f80ed28814ce6bL9